tokio/src/runtime/builder.rs

```diff
@@ -375,6 +385,30 @@ impl Builder {
             blocking_pool,
         })
     }
+
+    #[cfg(feature = "blocking")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "blocking")))]
+    /// Sets a custom timeout for a thread in the blocking pool.
+    ///
+    /// By default, the timeout for a thread is set to 10 seconds. This can
+    /// be overriden using .blocking_keep_alive().
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use tokio::runtime;
+    /// # use std::time::Duration;
+    ///
+    /// # pub fn main() {
+    /// let rt = runtime::Builder::new()
+    ///     .blocking_keep_alive(Duration::from_millis(100))
+    ///     .build();
+    /// # }
+    /// ```
+    pub fn blocking_keep_alive(&mut self, duration: Duration) -> &mut Self {
```

Comment:
I would probably name this `thread_keep_alive`. WDYT? Looks great though! Thanks.

```suggestion
    pub fn thread_keep_alive(&mut self, duration: Duration) -> &mut Self {
```